integration-tests/src/test/java/tachyon/client/keyvalue/KeyValueStoresIntegrationTest.java

```diff
@@ -261,4 +268,146 @@ public void putKeyValueTooLargeTest() throws Exception {
     mThrown.expectMessage(ExceptionMessage.KEY_VALUE_TOO_LARGE.getMessage(keyLength, valueLength));
     mWriter.put(key, value);
   }
+
+  /**
+   * Creates a store with the specified number of key-value pairs. The key-value pairs are in the
+   * format specified in {@link #genBaseKey(int)} and {@link #genBaseValue(int)} with id starts
+   * from 0.
+   *
+   * The created store's size is {@link Assert}ed before return.
+   *
+   * @param size the number of key-value pairs
+   * @param pairs the key-value pairs in the store, null if you don't want to know them
+   * @return the URI to the store
+   * @throws Exception if any error happens
+   */
+  private TachyonURI createStoreOfSize(int size, List<KeyValuePair> pairs) throws Exception {
+    TachyonURI path = new TachyonURI(PathUtils.uniqPath());
+    KeyValueStoreWriter writer = sKVStores.create(path);
+    for (int i = 0; i < size; i ++) {
+      byte[] key = genBaseKey(i).getBytes();
+      byte[] value = genBaseValue(i).getBytes();
+      writer.put(key, value);
+      if (pairs != null) {
+        pairs.add(new KeyValuePair(key, value));
+      }
+    }
+    writer.close();
+
+    Assert.assertEquals(size, sKVStores.open(path).size());
+
+    return path;
+  }
+
+  private int getPartitionNumber(TachyonURI storeUri) throws Exception {
+    KeyValueMasterClient client =
+        new KeyValueMasterClient(ClientContext.getMasterAddress(), ClientContext.getConf());
+    return client.getPartitionInfo(storeUri).size();
+  }
+
+  /**
+   * Creates a store with the specified number of partitions.
+   *
+   * NOTE: calling this method will set {@link Constants#KEY_VALUE_PARTITION_SIZE_BYTES_MAX} to
+   * {@link Constants#MB}.
+   *
+   * @param partitionNumber the number of partitions
+   * @param keyValuePairs the key-value pairs in the store, null if you don't want to know them
+   * @return the URI to the created store
+   */
+  private TachyonURI createStoreOfMultiplePartitions(int partitionNumber,
```

Comment:
`createStoreOfMultiplePartitions` has a lot code similar to `createStoreOfSize`, can we consolidate the code (feel free to leave this as a TODO)?